src/coprocessor/metrics.rs

```diff
@@ -185,6 +185,12 @@ lazy_static! {
         .unwrap();
     pub static ref MEM_LOCK_CHECK_HISTOGRAM_VEC_STATIC: MemLockCheckHistogramVec =
         auto_flush_from!(MEM_LOCK_CHECK_HISTOGRAM_VEC, MemLockCheckHistogramVec);
+
+    pub static ref COPR_CACHE_EFFICIENCY_VEC: IntCounterVec = register_int_counter_vec!(
+        "corp_cache_efficiency",
```

Comment:
```suggestion
        "tikv_coprocessor_cache_efficiency",
```

A good metric name should follow "\<namespace\>\_\<subsystem\>\_\<name\>".
See https://prometheus.io/docs/practices/naming/#metric-names